### ISSUE TITLE
Accent Colours/Colors

### ISSUE THREAD

**Issue by TheKodeToad** [PrismLauncher/PrismLauncher#903]
### Role

i like pink and other colours which are not green (i think)!

### Suggestion

Allow the user to select whichever accent colour they please (along with presets)!

### Benefit

The user isn't stuck with green.

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

**Comment by Poopooracoocoo** [comment]
What's green? Are you referring to the icons?

**Comment by TheKodeToad** [comment]
> What's green? Are you referring to the icons?

On the widgets.
Screenshot (first thing I saw :P):
![image](https://user-images.githubusercontent.com/57493648/222893096-b74be3ac-a55c-47b1-974e-9811d564b7ab.png)

I'm guessing you're not using the light or dark theme, but instead system.

**Comment by 678435021** [comment]
Yes, I am referring to the accent color used on the widgets. And yes, I am using the system theme. I would like the option to choose my own accent color, rather than being limited to the default green. This would allow for more personalization and customization of the user interface. Additionally, having preset accent color options would make it easier for users to choose a color that complements their personal style and preferences.

**Comment by yavko** [comment]
Each operating system (Linux, macOS, Windows) has an accent color API, so that could be used instead of adding this in-app. Also, as I can see you're on GNOME, so you could also just change your QT accents using qt5ct, or a kvantum theme that uses different accents.

**Comment by TheKodeToad** [comment]
> Each operating system (Linux, macOS, Windows) has an accent color API, so that could be used instead of adding this in-app. Also, as I can see you're on GNOME, so you could also just change your QT accents using qt5ct, or a kvantum theme that uses different accents.

Prism has its own accent colour override. Anyway, i guess that could be used as well as an in-app picker (i think that may still be useful)!

**Comment by yavko** [comment]
> > Each operating system (Linux, macOS, Windows) has an accent color API, so that could be used instead of adding this in-app. Also, as I can see you're on GNOME, so you could also just change your QT accents using qt5ct, or a kvantum theme that uses different accents.
> 
> Prism has its own accent colour override. Anyway, i guess that could be used as well as an in-app picker (i think that may still be useful)!

I just use a kvantum theme, and everything is bypassed, plus a prism theme can also do this

**Comment by orowith2os** [comment]
This feature would be blocked on Linux until https://github.com/flatpak/xdg-desktop-portal/pull/815 is merged.

**Comment by TheKodeToad** [comment]
I didn't mean the OS accent necessarily

**Comment by 0xMRTT** [comment]
> I didn't mean the OS accent necessarily

But that would be way better if it's using the OS accent colour...

**Comment by RokeJulianLockhart** [comment]
> But that would be way better if it's using the OS accent colour...

It's surprisingly simple to do so, especially for Windows, per https://stackoverflow.com/a/58528170. It's also worth adhering to the system accent colour on most DEs, simply because they provide such superior controls to what https://github.com/PrismLauncher/PrismLauncher/ could implement natively:

*	Android 10 to 13: https://www.androidauthority.com/change-system-accent-color-android-10-1025163/
*	Windows XP to 11: https://support.microsoft.com/en-us/windows/change-colors-in-windows-d26ef4d6-819a-581c-1581-493cfcc005fe
*	KDE Plasma 5 to 6:
	![image](https://github.com/PrismLauncher/PrismLauncher/assets/42837531/07f83fd6-9611-4d98-a5ee-a6d30dbb60f3) ![image](https://github.com/PrismLauncher/PrismLauncher/assets/42837531/55915a82-695b-4012-9530-40df62bfbc58)

**Comment by orowith2os** [comment]
@rokejulianlockhart adhering to the DE's preferred accent color would just involve reading the plain sRGB value from the portal, not any DE-specific configs like you just showed.

**Comment by yavko** [comment]
> @rokejulianlockhart adhering to the DE's preferred accent color would just involve reading the plain sRGB value from the portal, not any DE-specific configs like you just showed.

In a perfect world yeah, but idk any DE that has it implemented

**Comment by orowith2os** [comment]
@yavko GNOME has PRs, KDE has one (maybe already merged), Budgie has it planned, Elementary has it implemented, and there are many more to come. 

There's no need to read toolkit- and DE-specific configs. It's more work for no reason.

**Comment by RokeJulianLockhart** [comment]
But what about Windows? I expect `explorer.exe`, its default DE, doesn't integrate with XDG portals.

**Comment by orowith2os** [comment]
Then you'd read the Windows APIs. But you don't need to do DE-specific hacks on Linux.

**Comment by yavko** [comment]
Windows has a accent color system as it's set in the settings, I assume it's either an API or a registry setting

**Comment by RokeJulianLockhart** [comment]
@yavko, it's provided via both.
